frontend/src/pages/dashboard/Exports.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from "react";
+import { Row, Col, List, Space, Button, Tag, Popover } from "antd";
+import {
+  FileExcelFilled,
+  InfoCircleOutlined,
+  DownloadOutlined,
+} from "@ant-design/icons";
+import { api } from "../../lib";
+import upperFirst from "lodash/upperFirst";
+
+const ItemDescription = ({ created, tags }) => {
+  return (
+    <Space direction="vertical">
+      <div>{created}</div>
+      <div>
+        Filters:{" "}
+        {tags.length ? (
+          tags.map((t, i) => (
+            <Tag
+              key={i}
+              icon={
+                <Popover title={upperFirst(t.q)} placement="topRight">
+                  <InfoCircleOutlined />
+                </Popover>
+              }
+            >
+              {upperFirst(t.o)}
+            </Tag>
+          ))
+        ) : (
+          <Tag className="dotted-tag" style={{ margin: 0 }}>
+            None
+          </Tag>
+        )}
+      </div>
+    </Space>
+  );
+};
+
+const Export = () => {
+  const [fileList, setFileList] = useState([]);
+  const [pendingFile, setPendingFile] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [downloading, setDownloading] = useState(null);
+  const [page, setPage] = useState(1);
+  const [loadMoreButton, setLoadMoreButton] = useState(true);
+
+  useEffect(() => {
+    api
+      .get("download/list")
+      .then((res) => {
+        setLoading(false);
+        setFileList(res.data);
+      })
+      .catch(() => {
+        setLoading(false);
+        setFileList([]);
+      });
+  }, []);
+
+  const pending = fileList.filter((item) => item.status !== "Done");
+
+  useEffect(() => {
+    if (pending.length && !pendingFile) {
+      setTimeout(() => {
```

Comment:
Hi @navins94 , 
Let's say the "Pending" status does not change after 3 seconds, does that mean we need to refresh the page?
Or we can use an interval (maybe every xx seconds) instead of a timeout?

Reply:
Looking into it